### ISSUE TITLE
Date of image

### ISSUE THREAD

**Issue by bostig** [enricofer/go2mapillary#10]
HI,

Is it possibile to add label with date of image taken?

Regards,
Bosti

**Comment by enricofer** [comment]
Hi Bosti,
I'm planning some plugin improvements, including filtering and image/sequence metadata visualization.
In the meanwhile you can get the image/sequence date looking into attribute table of Mapillary_images and Mapillary_coverage layers. In there you will find a "captured_at" field reporting the milliseconds unix timestamp. Unfortunately in QGIS there ins't a native expression function for converting timestamp to datetime format so you would have to define a custom expression function in python:
```
from datetime import datetime
@qgsfunction(args='auto', group='Custom')
def from_timestamp(value1, feature, parent):
    return datetime.fromtimestamp(value1)
```
otherwise you can convert it online with a dedicated webservice: https://www.unixtimestamp.com/

**Comment by enricofer** [comment]
http://plugins.qgis.org/plugins/go2mapillary/
Implemented in the last release 2.0 with info button
